packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.tsx

```diff
@@ -137,13 +161,12 @@ export class VaSegmentedProgressBar {
                           </span>
                         ) : null
                       }
-                      
                     </li>
                   ))}
                 </ol>
                 {
                   <div class="usa-step-indicator__header">
-                    <Tag class="usa-step-indicator__heading">
+                    <Tag class="usa-step-indicator__heading" aria-describedby={describedbyIds}>
```

Comment:
Is this for external IDs? Does this work with the shadow dom?

Reply:
It's for external IDs, yes. I don't know if they work with the shadow DOM or not.

Reply:
It should at least be tested with VoiceOver otherwise I'm not sure why we would be adding something that doesn't work.

A workaround we have done in other components is add the `message-aria-describedby` prop to pass custom text that would be read by a screenreader inside the component. Here is an example from `va-text-area`: https://github.com/department-of-veterans-affairs/component-library/pull/788/commits/b7092dda18532b51039152dfe34574025679d129

Reply:
Ryan took a look and decided we shouldn't need the aria-describedby here, so went ahead and removed it. All that's left now is just the useDiv flag.